docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -434,6 +436,9 @@ Services and service descriptions are managed with ``xrdsst service`` subcommand
 is performed with ``xrdsst service add-description``. Enabling of service descriptions is performed  with ``xrdsst service enable-description``.
 Adding access to services is performed  with ``xrdsst service add-access``. Service parameters are updated with ``xrdsst service update-parameters``.
 
+### 4.1 Internal TSL certificates management
```

Comment:
I think it would be better to place this into "4.8 Certificate management" as all the "cert" commands are already there